src/persist-client/src/inspect.rs

```diff
@@ -324,6 +333,139 @@ pub async fn unreferenced_blobs(
     Ok(unreferenced_blobs)
 }
 
+/// Manually completes all fueled compactions in a shard.
```

Comment:
since we split out `inspect` and `admin` cmds it seems odd to have this in `inspect.rs`

Reply:
I was trying to keep K V and KVTD_CODECS from leaking out of the mod 🙈 . done!